### DISCUSSION TITLE
FDS installation on Linux cluster

### DISCUSSION THREAD

**Discussion by Muthu-V** [firemodels/fds#13877] [Q&A]
Let me know if there is any additional setups that we need to follow while Installing FDS on Linux Distribution
< Rocky Linux Release 8.10>
Do you recommend to compile the FDS code with any specific compiler version  for optimization ?

**Comment by mcgratta** [comment]
If you plan to compile the code yourself, follow these [instructions](https://github.com/firemodels/fds/wiki/FDS-Compilation). If you plan to install the pre-compiled executables, here are some [notes](https://github.com/firemodels/fds/wiki/Installing-and-Running-FDS-on-a-Linux-Cluster) that might be a bit dated. In either case, let us know how it's going. We currently undergoing many important changes in how we build FDS.

**Reply by Muthu-V** [reply]
Thant you @mcgratta
